web/src/App.scss

```diff
@@ -1,3 +1,4 @@
 .App {
-  background-color: #f0f0f0;
+  background-color: #2f2f2f;
+  height: 100vh;
```

Comment:
Games outside of the app div will have a white background colour since body is white. Might be better to move this to index.scss and target body or remove the height: 100vh